packages/utils/src/lib/git.integration.test.ts

```diff
@@ -21,12 +29,20 @@ describe('git utils', () => {
 
     await git.add('README.md');
     await git.commit('Create README');
+
+    await git.checkout(['-b', 'feature-branch']);
+    await git.checkout(['master']);
   });
 
   afterAll(async () => {
     await rm(baseDir, { recursive: true, force: true });
   });
 
+  afterEach(async () => {
+    await rm(changesDir, { recursive: true, force: true });
+    await git.checkout(['master']);
+  });
```

Comment:
```suggestion
  afterEach(async () => {
    await git.checkout(['master']);
    await git.deleteLocalBranch('feature-branch')
  });
```

And then in `beforeEach` e.g.
```
await git.checkout(['-b', 'feature-branch']);
```
If you want to start in that branch.

Reply:
I thought I keep the checkout of the branch in the tests as there are just some of them.

Reply:
ok, for some reason i cant reset a branch. added a todo and made it with manual deletion so far